allennlp/training/trainer.py

```diff
@@ -360,7 +360,27 @@ def _train_epoch(self, epoch: int) -> Dict[str, float]:
         logger.info("Training")
 
         cumulative_batch_group_size = 0
+        stopped_early = False
```

Comment:
This is a nit to pick, but I don't like the name `stopped_early`. "Early stopping" is something else.

Reply:
Heh, I originally called it `exit_early`, but that seemed worse. I will happily name it anything. I'll try `done_early`, but let me know if that's also bad...

Reply:
And fixed.